test/Makefile

```diff
@@ -0,0 +1,10 @@
+TEST = 1
+INCLUDE_UTILS = 0
+CFILES := ../src/infoPanel/imagesCache.c
+include ../src/common/config.mk
+
+TARGET = test_infoPanel
```

Comment:
How are we supposed to use this for multiple tests?

Reply:
Lets rename the target to something abstract

Reply:
Oh okay, so this Makefile is for building all tests?

Reply:
Yes, correct